Signal/src/ViewControllers/ConversationView/Cells/OWSMessageFooterView.m

```diff
@@ -146,6 +148,19 @@ - (void)configureWithConversationViewItem:(ConversationViewItem *)viewItem hasSh
     }
 }
 
+- (void)animateSpinningIcon
+{
+    CABasicAnimation *animation;
+    animation = [CABasicAnimation animationWithKeyPath:@"transform.rotation.z"];
+    animation.toValue = @(M_PI * 2.0);
+    const CGFloat kPeriodSeconds = 1.f;
+    animation.duration = kPeriodSeconds;
+    animation.cumulative = YES;
+    animation.repeatCount = HUGE_VALF;
+
+    [self.statusIndicatorImageView.layer addAnimation:animation forKey:@"animation"];
```

Comment:
nit: Do we want a more distinct key name than "animation"?

Reply:
I think that only matters if you're using more than one animation on a given layer, since you can then manipulate them by name.

Reply:
Exactly - if we add another animation do you think you'll remember to come back and make this one distinct? Or will you again use a generic name and inadvertently clobber this one?